### ISSUE TITLE
Data Table Coding is Used in Layout Table

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#3416]
**Functionality:**
- Evaluation Report

**Issue:**
Layout tables contain data table markup such as TH tags, ID, scope attributes, or Summary. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 1.3.1.

![image.png](https://images.zenhubusercontent.com/610014e44c513c0bad9f811c/5e05fd43-8e39-40e5-a3b9-71c1bce895bf)

<table role="presentation" class="width-auto"><tr><th class="text-right width-auto padding-top-2 text-normal">Facility ID (ORISPL):</th>

Table has role of presentation but also TH cells. Change the TH cells to TD

**Suggested Resolution:**
Remove any table markup that should only be used in data tables.

**Comment by aprematta** [comment]
Testing complete, issue has been fixed
